css/index.css

```diff
@@ -143,4 +237,143 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+/* My code starts here */
+
+.navigation {
+    margin: 5px;
+    width: 100%;
+    text-align: right;
+    font-size: 20px;
+    padding: 20px;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.container {
+    width: 1000px;
+    margin: 0px auto;
+}
+
+.logo {
+    margin: 15px;
+    position: absolute;
+    padding: 10px;
+    left: 0;
+    /* top: 10; */
+    /* display:  */
+}
+
+.jumbo-image {
+    width: 1000px;
+    padding: 10px;
+    display: flex;
+    justify-content: center;
+}
+
+.text-container {
+    margin: 20px auto;
+    padding: 20px;
+}
+
+.boxes .box-1 {
```

Comment:
Great job selecting the correct boxes to change the color

Reply:
Thanks.